### PR TITLE
Properly fill in make_non_contiguous data for sizes that can't be mad…

### PR DESCRIPTION
**Pull request by gchanan**
…e contiguous.

### CODE REVIEW

**Comment by gchanan** [comment]
This is a silly error and should fix the mac tests.

File: torch/testing/__init__.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 def make_non_contiguous(tensor):
     if tensor.numel() <= 1:  # can't make non-contiguous
-        return tensor.new(tensor.size())
+        return tensor.new(tensor.size()).copy_(tensor)
```
Comment (colesbury) [inline torch/testing/__init__.py]: `tensor.clone()`?

**Comment by gchanan** [comment]
this doesn't seem sufficient to fix the mac issues yet.

**Comment by gchanan** [comment]
actually, it does fix it (my Mac install was messed up).

**Pull request merged**